provisioning/miniprov/hare_mp/main.py

```diff
@@ -287,11 +287,23 @@ def save(filename: str, contents: str) -> None:
         f.write(contents)
 
 
+def generate_config(path_to_cdf: str):
```

Comment:
`def generate_config(path_to_cdf: str) -> int:` please.

Reply:
Updated the `generate_config()`, returning None now.

Reply:
done